Draft coops scenario - Close to enemies (#151)
qwrtln/Homm3BG-mission-book: draft-scenarios/coops/close_to_enemies.tex

% !TeX spellcheck = en_US
\addscenariosection{1}{Cooperative Scenario}{Close to enemies}{\images/balistic.png}

\begin{multicols}{2}

\textbf{Author:} Kelghor

\textbf{Version:} 1.5

\textbf{Source:} \href{https://discord.com/channels/740870068178649108/1313651865648369725}{Archon Studio Discord}

> \textit{"Sir! They're here again!"}

Reviewer: ```suggestion
\textit{"Sir! The raiders are at our gates!"}
```

> \textit{"Sound the alarm, we'll repel them as always!" you command, and go to defend the city. Just as you have done many times before. Such is the hard life of the surrounding clans, with constant wars and skirmishes with nearby neighbors. Fortunately, your alliance is strong, so you know your back is covered.}

Reviewer: ```suggestion
\textit{``Sound the warhorn! We'll drive them back as always!'' you command, and go to defend the city. Just as you have done many times before. Such is the harsh life of the border clans, where wars and skirmishes with neighboring rivals never truly end. Fortunately, your alliance remains strong, and you know your back is covered.}
```

> \textit{Therefore, the time has come to end the long-standing wars and crush your enemies like a bug once and for all!}

Reviewer: ```suggestion
\textit{The time has come to end these long-standing wars and crush your enemies once and for all!}
```

IMO, "like a bug" doesn't fit with Heroes setting.

Author: Actually i added it cuz in HOMAM IV was that like "Ajwar will crush enemy like a bug." Little easter egg :)

Reviewer: Oh, please add a comment with this. It should be a note to translators.

\subsection*{\MakeUppercase{Scenario Length}}

This Scenario is played over 7 Rounds.

\subsection*{\MakeUppercase{Player Setup}}

> \textbf{Player Count:} 1-6

Reviewer: ```suggestion
\textbf{Player Count:} 1--6
```

\textbf{Starting Resources:} 14 \svg{gold}, 5 \svg{building_materials}, 2 \svg{valuables}

\textbf{Starting Income:} 10 \svg{gold}, 2 \svg{building_materials}, 0 \svg{valuables}

\textbf{Starting Units:}
\begin{itemize}
  \item  1 × A Few \svgunit{bronze} Units of your choice
\end{itemize}

\textbf{Town Buildings:} \svgunit{bronze} Dwelling.

> \textbf{Additional Bonus:}: Only base bonus by difficulty (see the main rules).
> 

Reviewer: ```suggestion
```
No need for this paragraph, if that's the case.

> \textbf{Map Tile Pool:} Take $P$ × random Far (II–III) map Tiles as shared pool. None of them must contain a settlement. ($P$ stands for the number of players)

Reviewer: ```suggestion
\textbf{Map Tile Pool:} Take $P$ × random Far (II–-III) Map Tiles to form a shared pool, where $P$ equals the number of players. None of these tiles may contain a Settlement.
```

\subsection*{\MakeUppercase{Map Setup}}

> Take the following Map Tiles and arrange them as shown in the Scenario map layout.
> Setup Far (II--III) Map Tiles with settlements of same Faction as players Towns as show a color Cube, or see example.

Reviewer: ```suggestion
Take the following Map Tiles and arrange them as shown in the Scenario map layout.
Place Far (II--III) Map Tiles with Settlements that match your Faction's Towns. Mark these with a colored Cube (see example).
```
Is that what you meant to say here?

Reviewer: BTW, those cubes are barely visible. Maybe you could highlight those tiles on the picture with matching colors?

Author: Yes i will do it after code revision :-)

\begin{itemize}
  \item $P$ × Starting (I) Map Tile.
  \item $P$ × Far (II--III) Map Tile with Settlement of same Faction as players Towns.
  \item 2$P$ × Far (II--III) Map Tile - must not contain Tiles with Settlements.

> \end{itemize}

Reviewer: @Dima793 Little help here?

Author: What problem is here? :-)

Reviewer: @Dima793 changed the formatting around this recently in other scenarios. He might have an idea how to keep it consistent.

Reviewer 2: Current version doesn't compile for me so can't check, but I have two options to suggest:

Reviewer 2: Option 1
```suggestion
\begin{itemize}
  \item P × Starting (I) Map Tile.
  \item P × Far (II--III) Map Tile with the Settlement Field of the same Faction as player's Town.
  \item 2P × Far (II--III) Map Tile without the Settlement Field.
\end{itemize}
```

Reviewer 2: Option 2
```suggestion
\begin{itemize}
  \item P × Starting (I) Map Tile
  \item 3P × Far (II--III) Map Tile, exactly P of which must contain a Settlement of the same Faction as player's Town
\end{itemize}
```

Reviewer 2: In either option you can remove the "of the same Faction as player's Town" condition from the tile list if you think it looks too bulky, and add it as a note below, similar to the note below map tile list in the Titans' Stronghold scenario. But it affects tile choice rather than tile placement, so I think having it said within the tile list (as it is now) would be more consistent with how such conditions are arranged in other scenarios.

> \textit{Optional: You can distribute random pieces and covered pieces on the map so that they match your Faction's basic terrain.}

Author: That players can play this optional rule, and covered tiles place that same faction terrain will close to faction town tile... 

And random tiles will be known which tile is which terrain  so when player explore tile from pool he will explore his terrain. Im not sure that your wording is better. 

word "distribute" fit good i think. 

But maybe there could be longer & clearer way how to say it.

In fact it means, when players want, they distribute map tiles in way, that each will have  same mines in his hero trace, cuz each terrain has all types of mines...

> \textit{Optional: You can distribute random pieces and covered pieces on the map so that they match your Faction's basic terrain.}

Reviewer: All right, well, it's still not entirely clear to me. Let me think and come up with something else.

> \textit{Optional: You can distribute random pieces and covered pieces on the map so that they match your Faction's basic terrain.}

Author: "Players can take map tiles of their Faction and place them on the map according to the template so that they are connected together. They keep the last Faction tile with them and can use it during exploration. This option leads to an even distribution of mines."

> \textit{Optional: You can distribute random pieces and covered pieces on the map so that they match your Faction's basic terrain.}

Reviewer: Oh, if your intention is to have a just and even distribution of mines, then this should be clearly stated at the beginning.
> Optional: For an even distribution of mines...

> \textit{Optional: You can distribute random pieces and covered pieces on the map so that they match your Faction's basic terrain.}

Reviewer: How about something like this?

_To ensure a balanced distribution of Mines, players may place Map Tiles matching their Faction onto the Map Template, ensuring all placed Tiles are connected. Each player keeps their final Faction Tile for use during exploration._

But honestly, I'm still not getting the idea of this rule. Examples shown in pictures follow this rule or not? Or it's impossible to tell?

> \textit{Optional: You can distribute random pieces and covered pieces on the map so that they match your Faction's basic terrain.}

Reviewer 2: > But honestly, I'm still not getting the idea of this rule

I assume there are two advantages of this optional rule if it's used:
1. Gameplay/balance — each player is guaranteed to have one gold mine, one crystal mine and one stone mine.
2. Aesthetic/thematic — each player moves through their faction's own terrain, and also is more likely to utilize faction-specific mechanics using related map locations.

> \textit{Optional: You can distribute random pieces and covered pieces on the map so that they match your Faction's basic terrain.}

Reviewer 2: > They [players] keep the last Faction tile with them and can use it during exploration.

In that case it's not a *shared* pool of tiles anymore.

> \textit{Optional: You can distribute random pieces and covered pieces on the map so that they match your Faction's basic terrain.}

Reviewer: OK, let me get this straight now. There is an option to make a shared pool of tiles without settlements. I take it that while discovering new tiles not yet on the map, you draw from the pool at random. This optional rule basically cancels the shared pool, and every player gets one tile native to the home terrain of their faction?

> \textit{Optional: You can distribute random pieces and covered pieces on the map so that they match your Faction's basic terrain.}

Reviewer 2: > This optional rule basically cancels the shared pool, and every player gets one tile native to the home terrain of their faction?

It affects non-pool tiles too. And yes, among other things, it also kinda makes the shared pool not random and actually more like 1 tile per player. But maybe there is still an option to explore other player's tiles theoretically. Might need @Kelghor to explain their vision further.

> \textit{Optional: You can distribute random pieces and covered pieces on the map so that they match your Faction's basic terrain.}

Author: Hi, Dima793 understund the purpose 100% it can be eastetic and balanced distribution of Mine.

If is used this option, random pool is out.

"OK, let me get this straight now. There is an option to make a shared pool of tiles without settlements. I take it that while discovering new tiles not yet on the map, you draw from the pool at random. This optional rule basically cancels the shared pool, and every player gets one tile native to the home terrain of their faction?"
Yes its true

> \textit{Optional: You can distribute random pieces and covered pieces on the map so that they match your Faction's basic terrain.}

Reviewer: In that case, I think it makes more sense to keep it in a single place, rather than introducing Map Tile Pool, only to cancel it somewhere else. How about describing it in the Map Tile Pool section only, stating that each player gets a Far Map Tile matching the home terrain of their Faction, and mentioning that OPTIONALLY players can form a shared pool, to spice things up?

> \textit{Optional: You can distribute random pieces and covered pieces on the map so that they match your Faction's basic terrain.}

Reviewer: Of course, I'd love to see the purpose, too. You could mention that the former guarantees balanced distribution of mines, while the latter adds more randomness.

> \textit{Optional: You can distribute random pieces and covered pieces on the map so that they match your Faction's basic terrain.}

Author: So reworked it, this is outdated, pls check new version and create new discussion?

\subsection*{\MakeUppercase{AI Hero Setup}}

> Make AI Card Deck for Settlement defenders.

Reviewer: ```suggestion
Prepare an AI Deck for Settlement defenders.
```

> \textbf{Settlement defenders Deck:} 1×Attack Card, 1×Defence Card, 1×Spell Card

Reviewer: ```suggestion
\textbf{Settlement defenders Deck:} 1 × Attack Card, 1 × Defense Card, 1 × Spell Card
```

> \textbf{Settlement defenders Spell Deck:} 1×Magic Arrow

Reviewer: ```suggestion
\textbf{Settlement defenders Spell Deck:} 1 × Magic Arrow
```

\subsection*{\MakeUppercase{Victory Conditions}}

> Conquer all Settlements by any player.

Reviewer: ```suggestion
The game ends when a player conquers all settlements.
```

> Conquer all Settlements by any player.

Reviewer: ```suggestion
Conquer all Settlements.
```
Since it's a coop, I guess the latter part is redundant?

\subsection*{\MakeUppercase{Defeat Conditions}}

> \begin{itemize}
>   \item Any player lose his town.
>   \item All settlements are not owned by any player at the end of 7th turn.
> \end{itemize}

Reviewer: ```suggestion
\begin{itemize}
  \item Any player loses their Town.
  \item At least one Settlement remains unowned by the end of \nth{7} Round.
\end{itemize}
```
Careful with game terms, as rounds are not player turns.

> \subsection*{\MakeUppercase{Additional Rules}}

Reviewer: Please move them below timed events.

> \begin{itemize}
>   \item At any time, Players can spend their Building Token to perform the same actions as if they visited a Trading Post.
>   \item When defending the town, and your Hero is not here, you can still use your Units (for free).
>   \item Each settlement is defend by clans Settlement defenders. (see below)
> \end{itemize}

Reviewer: ```suggestion
\begin{itemize}
  \item Players can spend their Build Token at any time to perform Trading Post actions.
  \item You can use your Units to defend your Town (without paying 8 \svg{gold}) even if your Hero is not present.
  \item Each Settlement is defended by Settlement Defenders from local clans (see below).
\end{itemize}
```

\subsection*{\MakeUppercase{Timed Events}}

>   \begin{itemize}

Reviewer: Please remove this nested itemize, there's no need for it. After you do so, we'll work on the text.

    \begin{itemize}
      \item  First wave of enemy clan armies attacks the towns. (see below)
      \item  Each player who defended his own town, during this Round can hire secondary Hero for 5 \svg{gold}, but you still have to use the population token.
    \end{itemize}
   \item \textbf{\nth{3} Round:}
    \begin{itemize}
      \item At start of the Round, each player shows up one Tile (Far (II--III) Map Tile. This Tile you can reveal freely. (Not matter if Tile is from pool or map template.)
    \end{itemize}
   \item \textbf{\nth{4} Round:}
    \begin{itemize}
      \item Second wave of enemy clan armies attacks the towns. (see below)
      \item Each player who defended his own town, search (3) the \textit{Artifact Deck}.
     \end{itemize}
   \item \textbf{\nth{6} Round:}
    \begin{itemize}
      \item Third wave of enemy clan armies attacks the towns. (see below)
      \item Each player who defended his own town, search (3) the \textit{Artifact Deck}.
    \end{itemize}
   \item \textbf{\nth{7} Round:}
    \begin{itemize}
      \item During this Round, Players can discard and draw Cards at the start of their turn twice rather than once.
    \end{itemize}
  \end{itemize}

> \subsection*{\MakeUppercase{Enemy's clans waves}}

Reviewer: ```suggestion
\subsection*{\MakeUppercase{Enemy clan waves}}
```

\begin{itemize}

>   \item If enemy's settlement is not conquered, then enemy clans armies are attacking your castle!

Reviewer: ```suggestion
  \item Unconquered enemy Settlements will send clan armies to attack your Town!
```

>   \item At the start of the each player's turn, the battle begins.

Reviewer: ```suggestion
  \item A battle occurs at the start of each player's turn.
```

>   \item Before attack, you can use your building, spell book and population tokens, and use Might and Magic Cards, but you can't move with Hero.

Reviewer: ```suggestion
  \item Before combat, you may use your Building, Spell Book, and Population Tokens, as well as Might and Magic Cards. However, you cannot move your Hero.
```
What do you mean by using Might and Magic Cards here?

Author: For example you can use  Legion arm artefact for decrease unit cost. Or play gold sack for money. (cuz at start you will draw new cards)

Reviewer: So basically, use instant cards? If that's the intention, then this sentence is redundant, because you can always use them outside combat. Or maybe mention instant cards explicitly?

>   \item Each player has to defend his own castle.

Reviewer: ```suggestion
  \item Each player must defend their own Town.
```

>   \item If you have citadel, \textit{Arrow tower} has decreased attack. The attack is for first/second/third wave as 2/3/4\svg{attack}.

Reviewer: ```suggestion
  \item If you have a Citadel, the \textit{Arrow Tower} attack is reduced. The attacks for first/second/third wave become 2/3/4 \svg{attack}.
```

>   \item The battle does not cost any extra movement points.

Reviewer: ```suggestion
  \item This combat does not cost movement points.
```

>   \item The strength of the siege army is determined by the table \textit{Enemy's Clans armies}.

Reviewer: ```suggestion
  \item Refer to the \textit{Enemy Clan Armies} table for siege army strength.
```
You might want to add a hyperlink here.

>   \item If your main Hero is present in any of allied town (not settlement) and Clan wave attack your allied town where is no main Hero, you can participate on defending allied castle by using Might and Magic Cards (Simply: you defend your teammate's castle with their Units and your Hero's Cards).

Reviewer: ```suggestion
  \item If your main Hero is in an allied Town (not Settlement) when a Clan attacks another allied Town without a main Hero present, you may help defend using your Might and Magic Cards. You will use their Units and your Hero's Deck.
```

Author: Ah, i see bad wording now... 

It seems to be suposed,  that defending hero  control defending units... But there was supposed to   Defending player control their units and defending hero(player) can help them by using cards.

Reviewer: In that case
```suggestion
  \item If your main Hero is in an allied Town (not Settlement) when a Clan attacks another allied Town without a main Hero present, you may help defend using your Might and Magic Cards. The allied Hero will use their Units, and you can help them by playing Cards from your hand during combat.
```
But what happens if that combat is lost? What should happen to your main Hero that was helping defend the allied Town?

Reviewer: Oh, sorry, that would mean the scenario is lost. No need to add anything else.

>   \item If you defended your town, gain one \svg{experience}. It doesn't matter if the main Hero was in the fight.

Reviewer: ```suggestion
  \item Gain one \svg{experience} when successfully defending your Town, regardless of whether your main Hero participated.
```

\end{itemize}

\subsection*{\MakeUppercase{Settlements defenders}}
\begin{itemize}

>   \item The strength of the army in Settlement is determined by the table \textit{Enemy's Clans armies} (last row).

Reviewer: ```suggestion
  \item The strength of Settlement Defenders is determined by the \textit{Enemy Clan Armies} table (last row).
```

>   \item After draw an enemy army, Player can discard one enemy Unit Card and draw a new Unit of same Tier.

Reviewer: ```suggestion
  \item After drawing an enemy army, a Player may discard one enemy Unit Card and draw a new Unit of the same Tier.
```

>   \item The battle does not cost any extra movement points.

Reviewer: ```suggestion
  \item The combat does not cost Movement Points.
```

>   \item In this battles, you has to use AI Cards.

Reviewer: ```suggestion
  \item In these combats, you must use AI Cards.
```

>   \item After won battle, you gain one Level.

Reviewer: ```suggestion
  \item After winning a combat, gain one Level.
```
What exactly do you mean by that? Let's say your hero has 5 exp, meaning they're at level 2, need one experience to advance to level 3. On the other hand, having 4 exp would also advance them to level 3? Depending on the situation, the hero gets either 1 or 2 experience points?

Reviewer: ```suggestion
  \item After winning a combat, gain 2 \svg{experience}.
```
In that case, please make it precise. The original rule book is not reliable in this case.

\end{itemize}

\columnbreak

\vspace*{\fill}

\end{multicols}

\vspace{2em}

\begin{center}
  \hommtable[]{16}{
    \centering
    \medskip

>     \textbf{Enemy's Clans armies}\\

Reviewer: ```suggestion
    \textbf{Enemy's Clans armies}\\\\
```

Reviewer: To make it consistent with the text.

    \newcommand{\bronze}[0]{\svgunit[12]{bronze}}
    \newcommand{\silver}[0]{\svgunit[12]{silver}}
    \newcommand{\golden}[0]{\svgunit[12]{golden}}
    \newcommand{\azure}[0]{\svgunit[12]{azure}}

>       \begin{tabularx}{\linewidth}{p{0.2\linewidth}XXXX} & \darkcell{\bronze  Easy} & \darkcell{\silver  Normal} & \darkcell{\golden  Hard} & \darkcell{\azure  Impossible}\\

Reviewer: ```suggestion
      \begin{tabularx}{\linewidth}{p{0.2\linewidth}XXXX} &
      \darkcell{\raisebox{-.2\height}{\svgunit[20]{bronze}} \textbf{Easy}} &
      \darkcell{\raisebox{-.2\height}{\svgunit[20]{silver}} \textbf{Normal} &
      \darkcell{\raisebox{-.2\height}{\svgunit[20]{golden}} \textbf{Hard}} &
      \darkcell{\raisebox{-.2\height}{\svgunit[20]{azure}} \textbf{Impossible}}\\

```

      \darkcell[1.2]{First wave}
        & \lightcell[1.2]{\bronze}
        & \lightcell[1.2]{\bronze \bronze}
        & \lightcell[1.2]{\bronze \bronze \bronze}
        & \lightcell[1.2]{\bronze \bronze \silver} \\

      \darkcell[1.2]{Second wave}
        & \lightcell[1.2]{\bronze \bronze}
        & \lightcell[1.2]{\bronze \bronze \silver}
        & \lightcell[1.2]{\bronze \silver \silver}
        & \lightcell[1.2]{\silver \silver \silver} \\

      \darkcell[1.2]{Third wave}
        & \lightcell[1.2]{\bronze \silver \silver}
        & \lightcell[1.2]{\bronze \bronze \silver \silver}
        & \lightcell[1.2]{\bronze \bronze \silver \silver \silver}
        & \lightcell[1.2]{\bronze \bronze \silver \silver \golden} \\

      \darkcell[1.2]{Settlement defenders}
        & \lightcell[1.2]{ \bronze \silver \silver \golden \golden}
        & \lightcell[1.2]{ \silver \golden \golden \golden \golden}
        & \lightcell[1.2]{ \golden \golden \golden \golden \golden}
        & \lightcell[1.2]{ \silver \golden \golden \golden \azure} \\

      \end{tabularx}
  }
\end{center}

\vspace{2em}

\begin{center}
  \begin{figure}[h!]
    \centering

>     \captionof{figure}{\textbf{1-PLAYER SCENARIO | EXAMPLE }}

Reviewer: OK, I think I got it looking at the pictures.

>     \captionof{figure}{\textbf{1-PLAYER SCENARIO | EXAMPLE }}

Author: Yes, there is map tempalte and exmaple :-)

\textit{``Sir! The raiders are at our gates!''}

\textit{``Sound the warhorn! We'll drive them back as always!'' you command, and go to defend the city. Just as you have done many times before. Such is the harsh life of the border clans, where wars and skirmishes with neighboring rivals never truly end. Fortunately, your alliance remains strong, and you know your back is covered.}

\textit{The time has come to end these long-standing wars and crush your enemies like a bug once and for all!}
% Translator info: In Homam IV was Quote like: "Rufux will crush this army like a bug." If has scounting and has bigger army. So "like a bug" is little easter egg referencing on this.

\subsection*{\MakeUppercase{Scenario Length}}

This Scenario is played over 7 Rounds.

\subsection*{\MakeUppercase{Player Setup}}

\textbf{Player Count:} 1--6

\textbf{Starting Resources:} 14 \svg{gold}, 5 \svg{building_materials}, 2 \svg{valuables}

\textbf{Starting Income:} 10 \svg{gold}, 2 \svg{building_materials}, 0 \svg{valuables}

\textbf{Starting Units:}
\begin{itemize}
  \item  1 × A Few \svgunit{bronze} Units of your choice
\end{itemize}

\textbf{Town Buildings:} \svgunit{bronze} Dwelling.

> \textbf{Map Tile Pool:} Take $P$ × random Far (II–-III) Map Tiles to form a shared pool, where $P$ equals the number of players. None of these Tiles may contain a Settlement.

Reviewer: ```suggestion
\textbf{Map Tile Pool:} Take $P$ × random \mbox{Far (II--III)} Map Tiles to form a shared pool, where $P$ equals the number of players. None of these Tiles may contain a Settlement.
```
To get rid of artifacts and wierd splits.

\subsection*{\MakeUppercase{Map Setup}}

> Take the following Map Tiles and arrange them as shown in the Scenario map layout.
> Place Far (II--III) Map Tiles with Settlements that match your Faction's Towns. Mark these with a colored Cube (see example).
> 
> \begin{itemize}
>   \item P × Starting (I) Map Tile
>   \item P × Far (II--III) Map Tile with the Settlement Field of the same Faction as player's Town
>   \item 2P × Far (II--III) Map Tile without the Settlement Field
> \end{itemize}

Reviewer: ```suggestion
Take the following Map Tiles and arrange them as shown in the Scenario map layout:
\begin{itemize}
  \item P × Starting (I) Map Tile
  \item P × Far (II--III) Map Tile of your Town's home terrain that contains a Settlement
  \item 2P × Far (II--III) Map Tile without a Settlement
\end{itemize}
Place each Far (II--III) Map Tile containing a Settlement adjacent to the Starting (I) Map Tile of the same terrain type. Mark these Settlement tiles with a colored Cube (see example).
```
In the video game, Settlements belong to a faction, but in the adaptation this distinction has little sense. What these tiles have in common is terrain type. Also, current wording above the itemize list doesn't really explain what needs to be done. It is explained visually on the picture only.

Author: "Mark these Settlement tiles with a colored Cube (see example)."

This is not supposed to be marked by cube a tiles, but tiles marked by cube on map-template to know which far tiles with settlemens are where.

I think each owner knows wihch settlement is which faciton :D..  and it can be found in rules too (components of each faction),  If somebody do mistake with this, its not a deal for disbalance of scenario.  So i would keep that Faction wording. Cuz its lead to no question. Terrain wording leads to question about castle vs rampart settlements.

Reviewer: Ah, right, so this cube thing could be explained differently. I seriously thought that you're supposed to put them on the undiscovered tiles. Inferno campaign 3rd scenario does this very well: https://github.com/qwrtln/Homm3BG-mission-book/blob/main/assets/maps/inferno_steadwicks_fall.png

Also, I disagree with the faction thing making sense, as it makes sense only to people who played the video game, and can recognize those settlements visually. Many people asked on BGG and Discrod about Tower's settlement (not shown in the rule book), because they had no idea what it was. But it's your call, we can leave it at that.

Author: Im thinking, that in this version of scenario is not needed that rule... Only if they will use optional rule for same terrian. 

But its not matter, what settlment is where (in that case that you dont need conquer your faction settlement)

Author: So reworked it, this is outdated, pls check new version and create new discussion?

> \subsection*{\MakeUppercase{Map Setup}}
> 
> Take the following Map Tiles and arrange them as shown in the Scenario map layout.
> 
> \begin{itemize}
>   \item P × Starting (I) Map Tile
>   \item P × Far (II--III) Map Tile with the Settlement Field of the same terrain as player's Town.
>   \item 3P × Far (II--III) Map Tile without the Settlement Field. Each player takes three Map Tiles of their Faction's terrain with different mines. Each player place two of them on the game board so that they correspond with the map template and are linked together. Each player keeps the last Tile for personal exploration.
> \end{itemize}
> 
> Place Far (II--III) Map Tiles with Settlements that match your Faction's terrain, ensuring they align with the positions on the map represented by colored Cubes. Each pair of same-colored Cubes represents a Map Tile (I) and its corresponding Far (II--III) Map Tile with a Settlement.
> Each player place
> 
> \textit{Optional: If you want to spice up the game and deepen the cooperation between players, you can take all Far (II--III) Map Tile without the Settlement Field and place them on map randomly. Create a shared pool for discovery from the remaining Tiles.}
> 

Reviewer: OK, I had a thought about this and came up with the following. I think `Map Tile Pool` should not be omitted. The whole random/non-random rule is a place for just that. Then, info in the `Map Setup` can be shortened.

It boils down to describing the map setup in the map setup only, and all the things with map tile pool, either shared/random or not, in the map tile pool. Wanted to make it short and sweet, without too much text. I hope it captures the idea well. What do you think?

```suggestion
\textbf{Map Tile Pool:} Each player takes 1 random Far (II--III) Map Tile.
These Tiles must not contain Settlement Fields.

\textbf{Alternative Pool Setup (Optional):} To spice up the game and deepen cooperation between players, gather all Far (II--III) Map Tiles without Settlement Fields into a shared pool for exploration.

\subsection*{\MakeUppercase{Map Setup}}

Take the following Map Tiles and arrange them as shown in the Scenario map layout:

\begin{itemize}
 \item \textbf{P × Starting (I) Map Tile}
 
 \item \textbf{P × Far (II--III) Map Tile with Settlement}
 Place these Tiles according to the colored cube pairs shown on the map template.
 Each pair indicates positions for a Starting Tile and its matching Settlement Tile.
 
 \item \textbf{2P × Far (II--III) Map Tile without Settlement}
 Players place their two Tiles on designated spots.
 These Tiles must be connected to each other.
\end{itemize}
```

Author: I don't know where to start.

The current version seems unnecessarily overcomplicated. The fact that the Map tile pool has to be mentioned in its own section is unnecessarily constraining when it's an alternative method.
However, right now it's extra iteratively imprecise. 

"To spice up the game and deepen cooperation between players, gather all Far (II--III) Map Tiles without Settlement Fields into a shared pool for exploration."
Is not accrue. real deepen of cooperation is not in due shared map tile pool, but due not symmetrical distribution of mines. 
- It can easily happen that one player will have gold mines and other players will have crystals and stones, so then they have to either travel with the pieces or send resources to each other.

The crucial difference is that each player will either use their faction map-tiles or not. 
It's really about factional distribution of pieces, since each faction has its mines the same. It's about aesthetics and also making players less need to co-opt each other.

So either just have randomly distributed tiles in a map-template and a shared pool (needing more co-op)
Or for aesthetics/simplification -> for newbies for example. They each have their own color in their part of the territory in the map-template + 1 of their own tiles in hand to discover...
So each player takes 4 of his map tiles with one set element, and has 3 pieces left. Two he puts on the map in the map-template and the last one he doesn't want to discover. (So wherever he goes there will be waiting for him a possible undiscovered piece , with the mine that the player will be collecting.

I don't think it's necessary to explain the impact of the alternative fire. But rather to write that one is more coherent to cooperation and the other less so.

To me, it might be best to have map-tile pool + mapsetup as one variant, AND then the discovery section repeated for the other variant. 

Which variant will be the alternative doesn't really matter. 

Why am I sticking with the fractional map-tile? Because the 1st one will contain all the information (correct terrain color and content down). It can be more complicated to cast, I'd probably rather put that as a hint/tip or add the code of the map-tiles that belong to each faction in the notes.

One last thing wording this - I'm not sure if it's overkill.
" These Tiles must be connected to each other."

When I translate it, it looks to me like all tiles must be connected to each other, i.e. A+B, B+C but also A+C. However, it is enough if A+B,B+C,C+D are connected, but there is no need for C+A or D+A.

- I have no idea how to write it.

Take bych to rád nějak zjendodušil a víc rozdělil ty dvě možnosti.  Aby byl map pool a map setup víc provázané. 

Osobně sekce map-pool mi přidje strašně divná a nechápu proč je to vlastní sekce, když to jendoznačně patří do map-setupu. Ale to je fail už originálích scénářů.

PS: I Wish marry xmas. :-)

> \subsection*{\MakeUppercase{Map Setup}}
> 
> Take the following Map Tiles and arrange them as shown in the Scenario map layout.
> 
> \begin{itemize}
>   \item P × Starting (I) Map Tile
>   \item P × Far (II--III) Map Tile with the Settlement Field of the same terrain as player's Town.
>   \item 3P × Far (II--III) Map Tile without the Settlement Field. Each player takes three Map Tiles of their Faction's terrain with gold Mine, crystal Mine and one with anything. Each player shuffle their three Map Tiles and place two of them on the game board so that they correspond with the map template and are linked together. Each player keeps the last Tile for personal exploration.
> \end{itemize}
> 
> Place Far (II--III) Map Tiles with Settlements that match your Faction's terrain, ensuring they align with the positions on the map represented by colored Cubes. Each pair of same-colored Cubes represents a Map Tile (I) and its corresponding Far (II--III) Map Tile with a Settlement.
> Each player place
> 
> \textit{Optional: If you want to spice up the game and deepen the cooperation between players, you can take all Far (II--III) Map Tile without the Settlement Field shuffle them and place them on map randomly lead by map-template. After that, create a shared pool for discovery from the remaining Tiles.}

Reviewer: ```suggestion
\subsection*{\MakeUppercase{Map Setup}}

Take the following Map Tiles and arrange them as shown in the Scenario map layout:

\begin{itemize}
  \item P × Starting (I) Map Tile
  \item 3P × Far (II--III) Map Tile of the same terrain as player's Town, one of which must contain a Settlement
\end{itemize}

Place Far (II--III) Map Tiles with Settlements matching your Faction's terrain in positions marked by colored Cubes in the Scenario map layout.
Each pair of same-colored Cubes indicates a Starting (I) Map Tile and its corresponding Far (II--III) Map Tile with a Settlement.

\textbf{Map Tile Pool (choose one):}
\begin{itemize}
  \item \textbf{Personal Pool:} Each player takes 1~×~Far (II--III) Map Tile without a Settlement of the same terrain as player's Town
  \item \textbf{Shared Pool:}
  \begin{itemize}
    \item Using the Map Tiles that would form personal pools, create a shared discovery pool
    \item Remove all Far (II--III) Map Tiles without Settlements from the Map
    \item Shuffle all collected Map Tiles together
    \item Return Map Tiles without Settlements to the Map in random order
  \end{itemize}
\end{itemize}
```

I think I managed to grasp the idea now. This version lays out everything step-by-step, in simple bullet points. Firstly, you form the map, then you proceed to form the map tile pool, with two options laid out clearly. The formatting also turned out to be fitting:

![close-05](https://github.com/user-attachments/assets/1810d309-335d-4330-b216-d98eaa85cf42)

WDYT?

\subsection*{\MakeUppercase{AI Hero Setup}}

Prepare an AI Deck for Settlement defenders.

\textbf{Settlement defenders Deck:} 1 × Attack Card, 1 × Defense Card, 1 × Spell Card

\textbf{Settlement defenders Spell Deck:} 1 × Magic Arrow

\subsection*{\MakeUppercase{Victory Conditions}}
Conquer all Settlements.

\subsection*{\MakeUppercase{Defeat Conditions}}

\begin{itemize}
  \item Any player loses their Town.
  \item At least one Settlement remains unclaimed by the end of the \nth{7} Round. Time has run out!
\end{itemize}

\subsection*{\MakeUppercase{Timed Events}}

>     \textbf{\nth{1} Round:}
>     \begin{itemize}
>       \item At start of the Round, each player show up the \textit{nemesis map Tiles} (Far (II--III) Map Tile with Settlement of same Faction as players Towns). This Tile you can reveal freely.

Reviewer: ```suggestion
  \textbf{\nth{1} Round:}
  \begin{itemize}
    \item At the start of each Round, all players reveal their \textit{Nemesis Map Tiles} (Far Map Tiles (II--III) containing Settlements that match their Town's home terrain). These Tiles may be revealed without any cost.
```

Author: I has to disagree with this nnew wording. 

At the start of  **THE** Round, all players reveal their \textit{Nemesis Map Tiles} (Far Map Tiles (II--III) containing Settlements that match their Town's home terrain). These Tiles may be revealed without any cost.

not eahc round, only first round.. maybe you wanted say "**each Turn**"?

Reviewer: Yes, sorry, fixed.

>     \end{itemize}
>     \textbf{\nth{2} Round:}
>     \begin{itemize}
>       \item  First wave of enemy clan armies attacks the towns. (see below)
>       \item  Each player who defended his own town, during this Round can hire secondary Hero for 5 \svg{gold}, but you still have to use the population token.

Reviewer: ```suggestion
\end{itemize}
\textbf{\nth{2} Round:}
\begin{itemize}
  \item The first wave of Enemy Clan armies attacks all Towns (see section below).
  \item Each player who successfully defended their Town may hire a Secondary Hero for 5 \svg{gold} during this Round. This still requires spending a Population Token.
```

>     \end{itemize}
>     \textbf{\nth{3} Round:}
>     \begin{itemize}
>       \item At start of the Round, each player shows up one Tile (Far (II--III) Map Tile. This Tile you can reveal freely. (Not matter if Tile is from pool or map template.)

Reviewer: ```suggestion
\end{itemize}
\textbf{\nth{3} Round:}
\begin{itemize}
  \item At the start of each Round, each player reveals one Far Map Tile (II--III). This Tile may be revealed at no cost, whether it comes from the Pool or Map Template.
```

Author: same here

Reviewer: Fixed. I wonder if those "At the start of the Round" are necessary. Those events happen at the start anyway?

Author: I think its clearier that on a start of round not start of each players turn... its minor think but for players is better to explore it at start to more posibilities.

>     \end{itemize}
>     \textbf{\nth{4} Round:}
>     \begin{itemize}
>       \item Second wave of enemy clan armies attacks the towns. (see below)
>       \item Each player who defended his own town, search (3) the \textit{Artifact Deck}.

Reviewer: ```suggestion
\end{itemize}
\textbf{\nth{4} Round:}
\begin{itemize}
  \item The second wave of Enemy Clan armies attacks all Towns (see section below).
  \item Each player who successfully defended their Town may Search (2) the \textit{Artifact Deck} three times.
```
You mean regular Search (2) action here?

>      \end{itemize}
>     \textbf{\nth{6} Round:}
>     \begin{itemize}
>       \item Third wave of enemy clan armies attacks the towns. (see below)
>       \item Each player who defended his own town, search (3) the \textit{Artifact Deck}.

Reviewer: ```suggestion
\end{itemize}
\textbf{\nth{6} Round:}
\begin{itemize}
  \item The second wave of Enemy Clan armies attacks all Towns (see section below).
  \item Each player who successfully defended their Town may Search (3) the \textit{Artifact Deck}.
```

>     \end{itemize}
>     \textbf{\nth{7} Round:}
>     \begin{itemize}
>       \item During this Round, Players can discard and draw Cards at the start of their turn twice rather than once.
>     \end{itemize}

Reviewer: ```suggestion
\end{itemize}
\textbf{\nth{7} Round:}
\begin{itemize}
  \item During this Round, each player may perform the discard and draw action twice at the start of their turn, instead of the usual once.
\end{itemize}
```

\subsection*{\MakeUppercase{Additional Rules}}

\begin{itemize}
  \item Players can spend their Build Token at any time to perform Trading Post actions.
  \item You can use your Units to defend your Town (without paying 8 \svg{gold}) even if your Hero is not present.
  \item Each Settlement is defended by Settlement Defenders from local clans (see below).
\end{itemize}

\subsection*{\MakeUppercase{Enemy clan waves}}
\begin{itemize}
  \item Unconquered enemy Settlements will send clan armies to attack your Town!
  \item A battle occurs at the start of each player's turn.
  \item Before combat, you may use your Building, Spell Book, and Population Tokens, as well as Might and Magic Cards. However, you cannot move your Hero.
  \item Each player must defend their own Town.
  \item If you have a Citadel, the \textit{Arrow Tower} attack is reduced. The attacks for first/second/third wave become 2/3/4 \svg{attack}.

>   \item This combat does not cost movement points.

Reviewer: ```suggestion
  \item This combat does not cost \svgeven{movement}.
```

  \item Refer to the \textit{Enemy Clan Armies} table for siege army strength.
  \item If your main Hero is in an allied Town (not Settlement) when a Clan attacks another allied Town without a main Hero present, you may help defend using your Might and Magic Cards. The allied Hero will use their Units, and you can help them by playing Cards from your hand during combat.
  \item Gain one \svg{experience} when successfully defending your Town, regardless of whether your main Hero participated.
\end{itemize}

\subsection*{\MakeUppercase{Settlements defenders}}
\begin{itemize}
  \item The strength of Settlement Defenders is determined by the \textit{Enemy Clan Armies} table (last row).
  \item After drawing an enemy army, a Player may discard one enemy Unit Card and draw a new Unit of the same Tier.

>   \item The combat does not cost Movement Points.

Reviewer: ```suggestion
  \item The combat does not cost \svgeven{movement}.
```

  \item The battle does not cost any extra movement points.
  \item In this battles, you has to use AI Cards.
  \item After won battle, you gain one Level.
\end{itemize}

\columnbreak

\vspace*{\fill}

\end{multicols}

\vspace{2em}

\begin{center}
  \hommtable[]{16}{
    \centering
    \medskip
    \textbf{Enemy's Clans armies}\\

    \newcommand{\bronze}[0]{\svgunit[12]{bronze}}
    \newcommand{\silver}[0]{\svgunit[12]{silver}}
    \newcommand{\golden}[0]{\svgunit[12]{golden}}
    \newcommand{\azure}[0]{\svgunit[12]{azure}}

      \begin{tabularx}{\linewidth}{p{0.2\linewidth}XXXX} &

>       \darkcell{\raisebox{-.2\height}{\svgunit[20]{bronze}} \textbf{Easy}} &
>       \darkcell{\raisebox{-.2\height}{\svgunit[20]{silver}} \textbf{Normal} &
>       \darkcell{\raisebox{-.2\height}{\svgunit[20]{golden}} \textbf{Hard}} &
>       \darkcell{\raisebox{-.2\height}{\svgunit[20]{azure}} \textbf{Impossible}}\\

Reviewer: ```suggestion
      \darkcell{\textbf{Easy}} &
      \darkcell{\textbf{Normal}} &
      \darkcell{\textbf{Hard}} &
      \darkcell{\textbf{Impossible}}\\
```
Sorry, a syntax error in my suggestion broke the build. On 2nd thought, though, I think it's better without those unit stars in the header row.

![image](https://github.com/user-attachments/assets/26789ed3-be8c-4e64-b1c3-3122da50cad8)

      \darkcell[1.2]{First wave}
        & \lightcell[1.2]{\bronze}
        & \lightcell[1.2]{\bronze \bronze}
        & \lightcell[1.2]{\bronze \bronze \bronze}
        & \lightcell[1.2]{\bronze \bronze \silver} \\

      \darkcell[1.2]{Second wave}
        & \lightcell[1.2]{\bronze \bronze}
        & \lightcell[1.2]{\bronze \bronze \silver}
        & \lightcell[1.2]{\bronze \silver \silver}
        & \lightcell[1.2]{\silver \silver \silver} \\

      \darkcell[1.2]{Third wave}
        & \lightcell[1.2]{\bronze \silver \silver}
        & \lightcell[1.2]{\bronze \bronze \silver \silver}
        & \lightcell[1.2]{\bronze \bronze \silver \silver \silver}
        & \lightcell[1.2]{\bronze \bronze \silver \silver \golden} \\

>       \darkcell[1.2]{Settlement defenders}

Reviewer: ```suggestion
      \darkcell[1.2]{Settlement Defenders}
```
